web-admin/src/routes/[organization]/-/users/+page.svelte

```diff
@@ -0,0 +1,206 @@
+  import { Plus } from "lucide-svelte";
+  import AddUsersDialog from "@rilldata/web-admin/features/organizations/users/AddUsersDialog.svelte";
+  import { useQueryClient } from "@tanstack/svelte-query";
+  import { eventBus } from "@rilldata/web-common/lib/event-bus/event-bus";
+  import { Search } from "@rilldata/web-common/components/search";
+
+  let userEmail = "";
+  let userRole = "";
+  let isSuperUser = false;
+  let isAddUserDialogOpen = false;
+  let searchText = "";
+
+  $: organization = $page.params.organization;
+  $: listOrganizationMemberUsers =
+    createAdminServiceListOrganizationMemberUsers(organization);
+  $: listOrganizationInvites =
+    createAdminServiceListOrganizationInvites(organization);
+
+  function coerceInvitesToUsers(invites: V1UserInvite[]) {
+    return invites.map((invite) => ({
+      ...invite,
+      userEmail: invite.email,
+      roleName: invite.role,
+    }));
+  }
+
+  $: usersWithPendingInvites = [
+    ...($listOrganizationMemberUsers.data?.members ?? []),
+    ...coerceInvitesToUsers($listOrganizationInvites.data?.invites ?? []),
+  ];
+
+  $: filteredUsers = usersWithPendingInvites.filter((user) =>
+    user.userEmail.toLowerCase().includes(searchText.toLowerCase()),
+  );
+
+  const queryClient = useQueryClient();
+  const currentUser = createAdminServiceGetCurrentUser();
+  const addOrganizationMemberUser =
+    createAdminServiceAddOrganizationMemberUser();
+  const removeOrganizationMemberUser =
+    createAdminServiceRemoveOrganizationMemberUser();
+  const setOrganizationMemberUserRole =
+    createAdminServiceSetOrganizationMemberUserRole();
+
+  async function handleCreate(
+    newEmail: string,
+    newRole: string,
+    isSuperUser: boolean = false,
+  ) {
+    try {
+      await $addOrganizationMemberUser.mutateAsync({
+        organization: organization,
+        data: {
+          email: newEmail,
+          role: newRole,
+          superuserForceAccess: isSuperUser,
+        },
+      });
+
+      await queryClient.invalidateQueries(
+        getAdminServiceListOrganizationMemberUsersQueryKey(organization),
+      );
+
+      await queryClient.invalidateQueries(
+        getAdminServiceListOrganizationInvitesQueryKey(organization),
+      );
+
+      userEmail = "";
+      userRole = "";
+      isSuperUser = false;
+      isAddUserDialogOpen = false;
+
+      eventBus.emit("notification", { message: "User added to organization" });
+    } catch (error) {
+      console.error("Error adding user to organization", error);
+      eventBus.emit("notification", {
+        message: "Error adding user to organization",
+        type: "error",
+      });
+    }
+  }
+
+  async function handleRemove(email: string) {
+    try {
+      await $removeOrganizationMemberUser.mutateAsync({
+        organization: organization,
+        email: email,
+        // TODO: what is the default value for keepProjectRoles?
+        // params: {
+        //   keepProjectRoles: false,
```

Comment:
qq: do we know if we strip away users' project roles when we remove an org user? cc: @AdityaHegde @begelundmuller